src/main/kotlin/dev/arbjerg/ukulele/audio/Player.kt

```diff
@@ -100,8 +107,15 @@ class Player(val beans: Beans, guildProperties: GuildProperties) : AudioEventAda
         queue.clear()
         player.stopTrack()
     }
+    
+    fun loop() {
```

Comment:
This function is unnecessary because, you should just directly access the variable from your new command